src/reducers/eventReducer.js

```diff
@@ -11,6 +13,15 @@ const makeSearchable = (event) => ({
     filterableIncidentId: event.incidentId.toString()
 })
 
+const pageArgs = {
+    defaultPage: 1,
+    defaultSortOrder: 'desc',
+    defaultSortBy: 'occurred',
+    defaultPer: 10,
```

Comment:
what does "Per" means in this context?

Reply:
Number of records per page. The naming convention comes from the pagination middleware we're using; I don't think we can change it.